### DISCUSSION TITLE
Questions about installation notes

### DISCUSSION THREAD

**Discussion by v1nc3n4** [yarnpkg/berry#4628] [Q&A] [answered]
Hi,

I'm a bit confused after reading the latest [installation notes](https://yarnpkg.com/getting-started/install), starting with statement "The preferred way to manage Yarn is through [Corepack](https://nodejs.org/dist/latest/docs/api/corepack.html)". I ask myself these questions:

- `corepack` is marked as "Stability: 1 - Experimental" in the Node.js documentation. This [section](https://nodejs.org/dist/latest-v16.x/docs/api/documentation.html#stability-index) also highlights it is not recommended in production environments. Can the latest Yarn Berry release be safely used in production environments when managed with corepack?
- I understand the statement as a recommendation, though it's not clear for me who is the author of this recommendation (Yarn team, Node.js team?), and the target. Moreover, no alternative is introduced. In the past, there were one or more sections about installing Yarn 1 globally, then enabling Yarn Berry, then selecting the relevant version for our project. All these sections have disappeared. What are the other options?

Thanks in advance for these clarifications!
Best regards

**Comment by RDIL** [comment, marked as answer]
1. Yes, Berry is stable enough for use in production.
2. Corepack is still marked as experimental probably because it is new.  I don't know the exact specifics but I believe the Node.js team has a labeling process for all the exposed APIs and such.  Perhaps someone like @aduh95 can shed a bit more light on the specifics on that front.  In practice, it should be stable enough to use though.

**Reply by aduh95** [reply]
It's experimental because we want to keep the possibility to land breaking changes in its API. That being said, the core functionality (download and use the version of Yarn defined in the `package.json`) is very unlikely to change at this point, so you may consider it's  stable enough for your use case. I personally have been using it on production for a few months now.

**Reply by v1nc3n4** [reply]
Hi @RDIL @aduh95 ,
Thank you for these helpful answers!
BR
